.travis.yml

```diff
@@ -0,0 +1,67 @@
+---
+language: node_js
+node_js:
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "6"
```

Comment:
Node 8 is probably what you'd like

Reply:
Yes, thought about it, but decided to stick with the default addon blueprint of the latest Ember CLI for now, and to not make this necessarily a breaking change.

But I am fine with making that change, either as part of this PR or a separate one!?

Also related to breaking changes: I didn't find any notice about supported Ember versions. As this adds the default ember-try scenarios testing back to 2.18, this would be the minimum supported version (or do we want to be more aggressive?). We should make this explicit in the Readme, and would have to make this a breaking change release anyway, right? 

Please tell me how you want to handle this, and I'll make the necessary updates.

Reply:
> Yes, thought about it, but decided to stick with the default addon blueprint of the latest Ember CLI for now, and to not make this necessarily a breaking change.

They have been updated only a few days ago 😬.

> But I am fine with making that change, either as part of this PR or a separate one!?

This one is fine IMHO

> and to not make this necessarily a breaking change.

Node 8 would be breaking... But Node 6 as well. Currently, there is no minimum Node version set (neither in package.json nor in CI).

I don't feel comfortable with using Node 6 as it's not maintained by the Node team anymore. And there are chances that we don't update this very soon...

Reply:
> We should make this explicit in the Readme,

We might want to add this to the README (taken from the blueprints):

```md
Compatibility
------------------------------------------------------------------------------

* Ember.js v3.4 or above
* Ember CLI v2.13 or above
* Node.js v8 or above
```